### DISCUSSION TITLE
Different PP values when looking details of submitted score in game and on lazer.ppy.sh site

### DISCUSSION THREAD

**Discussion by Tyfaspo** [ppy/osu#21885] [Q&A] [answered]
I don't know if it's supposed to be like that but in-game PP values differds from PP values on site. Here's screenshots
![osu_2022-12-28_18-20-57](https://user-images.githubusercontent.com/68697602/209811792-f1636cfa-a206-4d48-a367-5d867082c42d.jpg)
![image](https://user-images.githubusercontent.com/68697602/209811836-27392eaf-bb39-4948-99c5-190270399930.png)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/21815
